pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -558,7 +561,6 @@ func (cloud *CloudProvider) insertZonalDisk(
 		opName        string
 		gceAPIVersion = GCEAPIVersionV1
 	)
-
 	if multiWriter || containsBetaDiskType(hyperdiskTypes, params.DiskType) {
```

Comment:
So since the default right now is Beta API, should this logic also change? Do we still want to use V1 for PD disks?

Reply:
Depends on if we decide to move forward with default Beta API

Reply:
No change needed as we are not switching default to Beta